test/bn_spatial_test.cpp

```diff
@@ -1172,9 +1171,9 @@ struct batch_norm_spatial_driver : test_driver
 #endif
         // this->tolerance = 80;
         // Debug values
-        // std::fill(input.begin(), input.end(), 1);
-        // std::fill(scale.begin(), scale.end(), 1);
-        // std::fill(shift.begin(), shift.end(), 1);
+        /* std::fill(input.begin(), input.end(), 1);
+        std::fill(scale.begin(), scale.end(), 1);
+        std::fill(shift.begin(), shift.end(), 1); */
```

Comment:
[Recommendation]
```suggestion
#if SELF_EXPLAINING_MACRO_NAME
        std::fill(input.begin(), input.end(), 1);
        std::fill(scale.begin(), scale.end(), 1);
        std::fill(shift.begin(), shift.end(), 1);
#endif
```
Remove `// Debug values` comment, and add somewhere near beginning of the source
```С++
#define SELF_EXPLAINING_MACRO_NAME 0
```